services/frontend/src/lib/api/user/models/UserPermissions.ts

```diff
@@ -0,0 +1,7 @@
+export interface UserPermissions {
+    username: string;
+    profilepicture: string;
+    permissions: string[];
+}
+
+export const USER_PERMISSION_ADMIN = "admin";
```

Comment:
permission does not exist so comment out
```suggestion
// define permissions as constants and refer to them as such for readability
// export const USER_PERMISSION_ADMIN = "admin";
```